src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/vcf/VcfFuncotationFactory.java

```diff
@@ -333,10 +335,13 @@ private static Funcotation mergeDuplicateFuncotationFactoryVariant(final Funcota
         final LinkedHashSet<String> allFieldNames = funcotation1.getFieldNames();
         allFieldNames.addAll(funcotation2.getFieldNames());
 
-        final Map<String, Object> mergedFieldsMap = allFieldNames.stream()
-                .collect(Collectors.toMap(f -> f, f -> mergeFuncotationValue(f, funcotation1, funcotation2, VcfFuncotationFactory::renderFieldConflicts)));
```

Comment:
You could use the `Collectors.toMap()` overloaded method that takes a `Supplier` and pass in `LinkedHashMap::new`. The default uses `HashMap::new`. 

In this case, using streaming doesn't add much, but for longer chains of stream operations this would be a good change. Might be worth having a utility function in GATK to make it easy.

See https://docs.oracle.com/javase/8/docs/api/java/util/stream/Collectors.html#toMap-java.util.function.Function-java.util.function.Function-java.util.function.BinaryOperator-java.util.function.Supplier-

Reply:
Yeah, you could do that.  You end up with a funny unused merge function if you do because this is assuming that you've pre-uniqued the inputs so I figured I'd just switch it to a for loop.